File: ghis/ghislain-one-app/src/app/business/track.ts
import {Injectable, OnInit} from '@angular/core';
declare var $:any;
 

@Injectable()
export class Track {
    Id:number = 0;
    AlbumId:number = 0;
    SongName:string = null;
    Length:string = null;
    Bytes:number = 0;
    UnitPrice:number = 0;
}
 

File: ZoeHome/src/ZoeHome.Domain/Entities/Track.cs
namespace ZoeHome.Domain.Entities
{
    public class Track
    {
        public int AlbumId { get; set; }
        public int Bytes { get; set; }
        public int Id { get; set; }
        public string Length { get; set; }
        public string SongName { get; set; }
        public decimal UnitPrice { get; set; }

        public override string ToString()
        {
            return SongName;
        }
    }
}